javascript-source/commands/customCommands.js

```diff
@@ -398,6 +402,15 @@
             }
             message = $.replace(message, message.match(/(\(gameonly=.*\))/)[1], '');
         }
+        
+        if (message.match(/\(useronly=.*\)/g)) {
+            var user = message.match(/\(useronly=(.*)\)/)[1];
```

Comment:
Since you're matching here, you don't have to match the string again when replacing.

Reply:
so would that be the same for gameonly as i did it the same way as that one and it has that??

Reply:
Yes. Not sure why it was done like that to begin with.